src/shared/components/card/cardProduct/CardProduct.css.ts

```
@@ -37,13 +38,33 @@ export const imgSection = recipe({
+    loaded: {
+      true: { opacity: 1 },
+      false: { opacity: 0 },
+    },
+  },
+  defaultVariants: {
+    loaded: false,
+  },
+});
+
+export const skeleton = style({
+  position: 'absolute',
```

ESLint

[error] Replace `'absolute'` with `"absolute"`

(prettier/prettier)